go.mod

```diff
@@ -6,6 +6,6 @@ require (
 	github.com/bep/hugo-jslibs/alpinejs v0.5.14 // indirect
 	github.com/bep/hugo-jslibs/instantpage v0.0.0-20200822093604-7b6e0aaba587 // indirect
 	github.com/bep/hugo-jslibs/turbolinks v0.1.2 // indirect
-	github.com/bep/linodedocs v0.0.0-20201218014640-53ad752e0ac9
+	github.com/bep/linodedocs v0.0.0-20201220215401-da781bf59b68
```

Comment:
this brings the PR up to date with the latest changes to the linode-website-partials repo from last friday: https://github.com/linode/linode-website-partials/commits/master